src/models/helpers/flowControl.ts

```diff
@@ -0,0 +1,61 @@
+function containsName(
+  arg:
+  string
+  | { name: string, prefix?: boolean, suffix?: boolean }
+  | Array<
+  string
+  | { name: string, prefix?: boolean, suffix?: boolean }
+  >,
+  callingFunctionName?: string | null,
+): boolean {
+  const functionName = callingFunctionName || (new Error().stack || '').split('\n')[2].trim().match(/at (\S+)/)?.[1] || '';
```

Comment:
since this function is only used in the this file and the ```skipif``` function below runs the exact same logic to extract a name if one is not provided, I wonder if you could just use callingFunctionName as its passed in (especially since you test it for truthiness below)

(not that its doing harm, but its extra code that should technically have tests written for each branch)